### PR TITLE
Update tutorial for accuracy 

### PR DESCRIPTION
**Pull request by sreevani-ship-it**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Updated contents to match V7 updates.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by IgorMinar — APPROVED** [review]
minor changes requested

File: aio/content/tutorial/toh-pt0.md [hunk 1]
```diff
@@ -6,30 +6,42 @@
```
Comment (jenniferfell) [inline aio/content/tutorial/toh-pt0.md]: While we're here, please add intro text between "The Application Shell" and the first subheading. I put a suggestion in ESI google docs, my playground folder, Tutorial file, suggestion 1: https://docs.google.com/document/d/17IndOUfK57ia7qefiOKQh5HhTUtsGhTKrmMBHqAjWjo/edit#

File: aio/content/tutorial/toh-pt0.md [hunk 2]
```diff
@@ -6,31 +6,44 @@
...
   npm install -g @angular/cli
-</code-example>  
+</code-example>
 
-## Create a new application
+<div class="alert is-helpful">
 
-Create a new project named `angular-tour-of-heroes` with this CLI command.
+**Note:** Angular requires `Node.js`. This tutorial assumes that you are using
+the [npm client](https://docs.npmjs.com/cli/install) command line interface, which is installed with `Node.js` by default. For more information, see the [Getting Started](guide/quickstart) guide.
```
[the remark this answers is not in the document]
Comment (jenniferfell) [reply aio/content/tutorial/toh-pt0.md]: I put a suggestion in ESI google docs, my playground folder, Tutorial file, suggestion 2: https://docs.google.com/document/d/17IndOUfK57ia7qefiOKQh5HhTUtsGhTKrmMBHqAjWjo/edit#

File: aio/content/tutorial/toh-pt0.md [hunk 3]
```diff
@@ -40,7 +52,7 @@ Go to the project directory and launch the application.
   cd angular-tour-of-heroes
```
Comment (jenniferfell) [inline aio/content/tutorial/toh-pt0.md]: Technically, this isn't the project folder, it's the workspace root. 
I put a suggestion in ESI google docs, my playground folder, Tutorial file, suggestion 4: https://docs.google.com/document/d/17IndOUfK57ia7qefiOKQh5HhTUtsGhTKrmMBHqAjWjo/edit#

**Review by jenniferfell — CHANGES_REQUESTED** [review]

**Review by jenniferfell — APPROVED** [review]

**Pull request closed without merging**